lib/Screens/Search/search.dart

```diff
@@ -210,6 +211,7 @@ class _SearchPageState extends State<SearchPage> {
                                                     .toString()
                                                     .trim();
                                                 controller.text = query;
+                                                controller.selection = TextSelection.fromPosition(TextPosition(offset: query.length));
```

Comment:
Just a small formatting issue.
```suggestion
                                                controller.selection = TextSelection.fromPosition(
                                                	TextPosition(offset: query.length),
                                                );
```